src/components/network/network.js

```diff
@@ -541,71 +542,71 @@ export default class Network {
         }
     }
 
-    removeEquipment(equipmentType, equipmentId) {
-        switch (equipmentType) {
-            case 'LINE':
+    removeEquipment(feederType, equipmentId) {
+        switch (feederType) {
+            case FEEDER_TYPES.LINE.type:
                 this.lines = this.lines.filter((l) => l.id !== equipmentId);
                 this.completeLinesInfos();
                 break;
-            case 'TWO_WINDINGS_TRANSFORMER':
+            case FEEDER_TYPES.TWO_WINDINGS_TRANSFORMER.type:
```

Comment:
FEEDER_TYPES is only for SLD
Here this EQUIPMENTS_TYPES

Reply:
done